src/HaskellCI/GitHub.hs

```diff
@@ -556,16 +561,37 @@ cat path contents = sh $ concat
     , "EOF"
     ]
 
+-- |
```

Comment:
It might be worth mentioning here that this will likely not parse all possible Git URLs, but it should cover most common use cases.

Reply:
Git**Hub** urls.. :)

Reply:
I added a comment

Reply:
Yes, good point.